### DISCUSSION TITLE
API for creating tags, document_type and correspondent

### DISCUSSION THREAD

**Discussion by toushin-taishi** [jonaswinkler/paperless-ng#1199] [Feature Requests]
Hello,

As the title suggests, will it be possible for the future release of paperless-ng to include the capability of creating tags, document_type, and correspondent via API? 

Hoping for your favorable response.

Kind regards

**Comment by sbrunner** [comment]
Yes, the documentation is here: https://paperless-ng.readthedocs.io/en/latest/api.html
Currently, I have some difficulties to use it, see: https://github.com/jonaswinkler/paperless-ng/discussions/1413
